services/galley/test/integration/API.hs

```diff
@@ -1026,6 +1027,49 @@ putMemberOk update g b ca = do
         assertEqual  "hidden"           (memHidden memberBob)         (memHidden newBob)
         assertEqual  "hidden__ref"      (memHiddenRef memberBob)      (memHiddenRef newBob)
 
+putReceiptModeOk :: Galley -> Brig -> Cannon -> TestSetup -> Http ()
+putReceiptModeOk g b c _ = do
+    alice <- randomUser b
+    bob   <- randomUser b
+    jane  <- randomUser b
+    connectUsers b alice (list1 bob [jane])
+    cnv <- decodeConvId <$> postConv g alice [bob, jane] (Just "gossip") [] Nothing Nothing
+    WS.bracketR3 c alice bob jane $ \(_wsA, wsB, _wsJ) -> do
+        getConv g alice cnv !!! do
+            const 200 === statusCode
+            const (Just Nothing) === fmap cnvReceiptMode . decodeBody
+        -- Set receipt mode
+        put ( g
+         . paths ["conversations", toByteString' cnv, "receipt-mode"]
+         . zUser alice
+         . zConn "conn"
+         . zType "access"
+         . json (ConversationReceiptModeUpdate $ ReceiptMode 0)
+         ) !!! const 200 === statusCode
+        void . liftIO $ checkWs alice (cnv, wsB)
+        -- No changes
+        put ( g
+         . paths ["conversations", toByteString' cnv, "receipt-mode"]
+         . zUser alice
+         . zConn "conn"
+         . zType "access"
+         . json (ConversationReceiptModeUpdate $ ReceiptMode 0)
+         ) !!! const 204 === statusCode
+        -- Ensure that the new field is set
```

Comment:
- check that also after the first put?
- check that the WS doesn't have any noise after the second put?